test/e2e/fixture-builder.js

```diff
@@ -768,7 +768,7 @@ class FixtureBuilder {
                 maxFeePerGas: '0x59682f0c',
                 maxPriorityFeePerGas: '0x59682f00',
               },
-              id: 7911313280012623,
+              id: '7911313280012623',
```

Comment:
For the sake of readability, should we use UUIDs in any updated test data to avoid confusion down the line?

Reply:
True, this could be confusing for people without context, done in https://github.com/MetaMask/metamask-extension/pull/20952/commits/bf96a823f5f3e9cbbd392d226206cf6b43bc0ae4